server/src/test/resources/config/sidecar_multiple_instances.yaml

```diff
@@ -71,6 +71,9 @@ sidecar:
   tcp_keep_alive: false
   accept_backlog: 1024
   server_verticle_instances: 1
+  vertx_filesystem_options:
+    class_path_resolving_enabled: false
```

Comment:
I thought `classpath` is commonly recognized as a single word :p https://en.wikipedia.org/wiki/Classpath

Reply:
good call, I missed this